packages/gestalt-design-tokens/build.js

```diff
@@ -256,6 +297,8 @@ function getSources({ theme, modeTheme, platform }) {
     `tokens/vr-theme/base-elevation${modeTheme}.json`,
     'tokens/vr-theme/sema-elevation.json',
     `tokens/vr-theme/sema-color${modeTheme}.json`,
+    `tokens/vr-theme/base-line-height.json`,
+    `tokens/vr-theme/language/sema-line-height-${language}.json`,
```

Comment:
can we just inline the files here rather than putting them below in the forEach? line 1080

```

    `tokens/vr-theme/language/sema-line-height-default.json`,
    `tokens/vr-theme/language/sema-line-height-js.json`,
...

```

I think it's easier to follow fro here than all the drilling of the languages

Reply:
Agreed, inline would look cleaner. 

But I'm not able to because `sema-line-height-ja` exports the same tokens as `sema-line-height-default`, running it in the for-each loop as a separate build allows us to avoid the conflict

Otherwise, there are conflicts because the exported names are the same